content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -18,130 +18,115 @@ endif::[]
 
 = Pipeline Editor
 
-The Blue Ocean Pipeline Editor is the simplest way for anyone to get started with
-creating Pipelines in Jenkins. It's also a great way for existing Jenkins users
-to start adopting Pipeline.
+The Blue Ocean Pipeline editor is an easy way for anyone to get started with creating Pipelines in Jenkins.
```

Comment:
I don't know the Jenkins product abbreviations well, in case my suggestion doesn't apply, but pretty sure this is JOSS. 
```suggestion
The Blue Ocean Pipeline editor is an easy way for anyone to get started with creating Pipelines in {JOSS}.
```

Reply:
Hi @cari-mason no worries at all! In this case, those macros would not work correctly in the Jenkins docs, so every usage of Jenkins will be written out accordingly. I think I've run into this before, and was able to confirm that unfortunately these are custom macros for the CloudBees docsite repo specifically.